dubbo-rpc/dubbo-rpc-triple/src/main/java/org/apache/dubbo/rpc/protocol/tri/TripleInvoker.java

```diff
@@ -260,7 +265,10 @@ RequestMetadata createRequest(MethodDescriptor methodDescriptor, Invocation invo
         if (methodDescriptor instanceof PackableMethod) {
             meta.packableMethod = (PackableMethod) methodDescriptor;
         } else {
-            meta.packableMethod = ReflectionPackableMethod.init(methodDescriptor, url);
+            final String contentType = APPLICATION_GRPC + "+" + UrlUtils.serializationOrDefault(url);
```

Comment:
Do not dynamically concatenate content types for now

Reply:
Fixed